templates/README.md

```diff
@@ -52,24 +52,32 @@ metadata:
     # that will do the find and replace using a syntax documented
     # here: https://kubernetes.io/docs/reference/kubectl/jsonpath/
     # The jsonpath root is the objects: element of the template
+    # XXX There is currently an issue with annotations not supporting
+    #     list values
     template.cnv.io/replace:
     - path: "/VM/disks/*/type"
       value: virtio
 
     # Another extension useful for CNV aware tooling that allows
     # expressing additional validation rules for this template
     # The jsonpath root is the objects: element of the template
+    # XXX There is currently an issue with annotations not supporting
+    #     list values
```

Comment:
This should go in the commit message. Having to dig into the code to find out which are the issues that have to be fixed in the template is not so practical.

Also, couldn't this commit be squashed to the one adding the templates?